src/richie/apps/courses/migrations/0020_deprecate_course_glimpse_plugin_small_variant.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.3 on 2020-11-16 08:57
+
+from django.db import migrations
+
+from ..defaults import COURSE_GLIMPSE_VARIANT_CHOICES
+
+
+def forwards_func(apps, schema_editor):
+    """
+    Update every course plugin objects which used the deprecated "small" variant
+    to default course variant.
+    """
+    CoursePlugin = apps.get_model("courses", "coursepluginmodel")
+    CoursePlugin.objects.filter(variant="small").update(
+        variant=COURSE_GLIMPSE_VARIANT_CHOICES[0][0]
+    )
```

Comment:
Sorry I just realized that this may break the db for someone who has customized these choices
Maybe add an instruction to UPGRADE.md to make the necessary treatments themselves?